src/js/main.js

```diff
@@ -0,0 +1,59 @@
+const form = document.getElementById('registrationForm');
+const fullName = document.getElementById('fullName');
+const phoneNumber =  document.getElementById('phoneNumber');
+const hideInfo = form.querySelector('#hidden-info');
+const privateInfo = document.getElementById('privateInfo');
+const inputRegion = form.querySelector('#region');
+const city = form.querySelector('select#city');
+const cities = form.querySelector('#cities');
+const submit = form.querySelector('input#submit');
+
+fullName.addEventListener('blur', () => {
```

Comment:
not sure 'blur' is the best event to subscribe to. would smth like keyup/change work better?